src/reviewRenderer/renderers/interactions/PortableCustomInteraction.js

```diff
@@ -35,7 +35,7 @@ const getData = function (customInteraction, data) {
 
     //remove ns + fix media file path
     markup = util.removeMarkupNamespaces(markup);
-    markup = PortableElement.fixMarkupMediaSources(markup, this);
+    markup = PortableElement.fixMarkupMediaSources(markup, customInteraction.renderer);
```

Comment:
**thought:** The weird part was very similar to what we have for the regular mode as you can see there: https://github.com/oat-sa/tao-item-runner-qti-fe/blob/96d4fed783e9faccff9d8d226c4d7969cfa116b5/src/qtiCommonRenderer/renderers/interactions/PortableCustomInteraction.js#L192

`getData()` is a method that is expected to be called with the proper lexical `this`: https://github.com/oat-sa/tao-item-runner-qti-fe/blob/96d4fed783e9faccff9d8d226c4d7969cfa116b5/src/qtiRunner/core/Renderer.js#L360

Reply:
You are right! It was fixed with converting arrow function to normal function by Hanna: https://github.com/oat-sa/tao-item-runner-qti-fe/commit/65a9e31f83fb826440b69c653d0bb33994d1c1f8
I close this PR.